prac_05/wimbledon.py

```diff
@@ -0,0 +1,47 @@
+"""
+CP1404 - Prac 5
+Program: Wimbledon
+"""
+
+FILENAME = 'wimbledon.csv'
+COUNTRY_INDEX = 1
+CHAMPION_INDEX = 2
+
+def main():
+    """Load records from file and display details of champions."""
+    records = load_records()
+    champion_to_win_count, countries = process_records(records)
+    display_results(champion_to_win_count, countries)
+
+def load_records():
+    """Create a list of records using data from CSV file."""
+    records = []
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()
+        for line in in_file:
+            parts = line.strip().split(',')
+            records.append(parts)
+    return records
+
+def process_records(records):
+    """Create a dictionary of champions to win count."""
+    champion_to_win_count = {}
+    countries = set()
+    for record in records:
+        countries.add(record[COUNTRY_INDEX])
+        try:
+            champion_to_win_count[record[CHAMPION_INDEX]] += 1
+        except KeyError:
+            champion_to_win_count[record[CHAMPION_INDEX]] = 1
+    return champion_to_win_count, countries
+
+
+def display_results(champion_to_win_count, countries):
```

Comment:
The output format is clear and organized. Displaying each champion and their win count, followed by a list of countries, provides a logical breakdown of information.